celer/PN_logreg.pyx

```diff
@@ -161,8 +161,7 @@ def newton_celer(
             for i in range(n_samples):
                 theta_acc[i] = y[i] * sigmoid(- y[i] * theta_acc[i])
 
-            tmp = 1. / alpha
-            fscal(&n_samples, &tmp, &theta_acc[0], &inc)
+            tmp = 1.
```

Comment:
what is the use for this ?

Reply:
I forgot to remove it

Reply:
ok perfect, remove it and the bench file and we merge !